### DISCUSSION TITLE
Stuck with a homing problem

### DISCUSSION THREAD

**Discussion by Team-Deepflights** [terjeio/grblHAL#349] [Q&A]
I can't put limit switches at the zero position of the lathe, so I put them to the max ends, I thought I could change in the firmware to tell that endswitches are at max position, but I did not found anything like that. I know this from other controller software thats why I thought it's possible here too.
For me it's not a real problem that I have to go to -200 instead of 200 but my CAM-tool write the moves always in positive digits (for now I did not found any solution for the postprocessor)
Is it somewhere possible to tell Grbl that 0= max travel distance?
It would be easier than rewrite the whole postprocessor :-)

**Comment by terjeio** [comment]
$22:

![image](https://user-images.githubusercontent.com/20260062/148287736-68ef4aa6-d3eb-42d9-a768-989f24c49133.png)

and set G54 or G92 (with persistence enabled - $384) to match max travel?
